src/vm.zig

```diff
@@ -421,6 +421,7 @@ pub const VM = struct {
     current_fiber: *Fiber,
     current_ast: Ast.Slice,
     globals: std.ArrayListUnmanaged(Value) = .{},
+    global_names: std.AutoArrayHashMap(u24, []const u8),
```

Comment:
The hashmap should be `StringHashMap(u24)`: you want to be able to retrieve the global's index from its name